tasks/main.yml

```diff
@@ -20,7 +20,7 @@
     - npm
 
 - name: Determine what version of NPM is installed
-  command: npm --version
+  command: /usr/local/bin/npm --version
```

Comment:
I worry that this change makes things more brittle because it causes the path to be fixed. Before, if the path was different, it would still work as long as `npm` exists in the `PATH`.

This role places a symlink for the `node` and `npm` binaries into `/usr/local/bin`. I think a potential solution would be to add an additional variable (`node_path` or something) that is used to build the path to the binary referenced here, and is also used to produce the symlinks.